src/components/Header.js

```diff
@@ -11,10 +16,12 @@ const Header = (props) => {
 	return (
     <div style={styles.header__flex}>
       <div style={[styles.header]}>
-        <h1 style={styles.header__title} onClick={props.handleHideCurrencyList}>CryptoTraq</h1>
+        <h1 style={styles.header__title}>CryptoTraq</h1>
       </div>
       <div style={styles.header__button_box}>
-        <Button onClick={props.handleShowCurrencyList} buttonContent={"Supported currencies"}/>
+        <Button
+					onClick={props.handleToggleCurrencyView}
+					buttonContent={props.showingCurrencyList ? "Back to traq" : "Supported currencies"}/>
```

Comment:
I added a conditional statement to change the `buttonContent` based on which view the user was seeing.
- If the user is on the main view where they are selecting their currencies, then the button will display `Supported Currencies`
- If the user is on the Supported Currencies view, the button will display `Back to traq` and will toggle the main view to show again.